### PR TITLE
feat(contrib): kill containers on success or failure

### PR DESCRIPTION
**Pull request by bacongobbler**

### CODE REVIEW

File: contrib/ci/test.sh [hunk 1]
```diff
@@ -49,6 +54,9 @@ MINIO_JOB=$(docker run -dv $CURRENT_DIR/tmp/aws-admin:/var/run/secrets/deis/mini
 # boot postgres, linking the minio container and setting DEIS_MINIO_SERVICE_HOST and DEIS_MINIO_SERVICE_PORT
 PG_JOB=$(docker run -d --link $MINIO_JOB:minio -e BACKUP_FREQUENCY=1s -e DEIS_MINIO_SERVICE_HOST=minio -e DEIS_MINIO_SERVICE_PORT=9000 -v $CURRENT_DIR/tmp/creds:/var/run/secrets/deis/database/creds -v $CURRENT_DIR/tmp/aws-user:/etc/wal-e.d/env $1)
 
+# kill containers when this script exits or errors out
```
Comment (carmstrong) [inline contrib/ci/test.sh:57]: Should we also trap TERM here?

Comment (bacongobbler) [inline contrib/ci/test.sh:57]: After a bit of reading apparently it's better to use `trap ... INT TERM` instead for better portability across different shells. Good point.

Comment (bacongobbler) [inline contrib/ci/test.sh:57]: done!

Comment (carmstrong) [inline contrib/ci/test.sh:57]: This is what we did in v1: https://github.com/deis/deis/blob/master/database/bin/boot#L110-L116

**Pull request merged**